chat-plugins/othermetas.js

```diff
@@ -0,0 +1,134 @@
+// Other Metas plugin by Spandan
+'use strict';
+
+exports.commands = {
+	mnm: 'mixandmega',
+	mixandmega: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		if (!toId(target) || !target.includes('@')) return this.parse('/help mixandmega');
+		let sep = target.split('@');
+		let stone = toId(sep[1]);
+		let template = toId(sep[0]);
+		if (!Tools.data.Items[stone] || (Tools.data.Items[stone] && !Tools.data.Items[stone].megaEvolves && !Tools.data.Items[stone].onPrimal)) {
+			return this.errorReply(`Error: Mega Stone not found`);
+		}
+		if (!Tools.data.Pokedex[toId(template)]) {
+			return this.errorReply(`Error: Pokemon not found`);
+		}
+		template = Object.assign({}, Tools.getTemplate(template));
+		stone = Object.assign({}, Tools.getItem(stone));
+		if (template.isMega || (template.evos && Object.keys(template.evos).length > 0)) { // Mega Pokemon cannot be mega evolved
```

Comment:
This should have been `template.nfe` but I only noticed because the code has been removed so meh.